controller/api/destination/endpoint_translator.go

```diff
@@ -15,7 +17,12 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-const defaultWeight uint32 = 10000
+const (
+	defaultWeight uint32 = 10000
+	// inboundListenAddr is the environment variable holding the inbound
+	// listening address for the proxy container.
+	inboundListenAddr = "LINKERD2_PROXY_INBOUND_LISTEN_ADDR"
```

Comment:
nit: perhaps `inboundListenAddrEnv` to make it clearer that it's the env variable and not the addr itself

Reply:
I went with `envInboundListenAddr` since we use a similar naming [here](https://github.com/linkerd/linkerd2/blob/main/pkg/identity/service.go#L29)